includes/Modules/AdSense.php

```diff
@@ -72,14 +78,34 @@ final class AdSense extends Module
 	const MODULE_SLUG = 'adsense';
 
 	/**
-	 * Ad blocking recovery tag option name.
+	 * Ad_Blocking_Recovery_Tag instance.
+	 *
+	 * @since n.e.x.t
+	 * @var Ad_Blocking_Recovery_Tag
```

Comment:
```suggestion
	 * 
	 * @var Ad_Blocking_Recovery_Tag
```

Reply:
This is consistent with how it's done on the base `Module` class.

Reply:
Fair enough; feel free to leave as-is. We've never been entirely consistent about it. There are explicit rules that just haven't made it into PHPCS yet. See #924

> * For methods and functions, tags should be grouped: `since`/`deprecated`/`access`/`static` should be grouped together, separated by a blank line from `param`/`return` which should be grouped together. If there is `global`, it should be separated from both others by a blank line (essentially be its own group).

Regarding `var` it's order is defined as coming after `global` so it seems that it would be implicitly separated from the above grouping with `@since`.

Again, just for context; don't worry about it 😉